pkg/mmesh/modelmesh_service.go

```diff
@@ -89,6 +94,14 @@ func (mms *MMService) InferenceEndpoint() string {
 	return fmt.Sprintf("%s:%d", mms.Name, mms.Port)
 }
 
+func (mms *MMService) InferenceRESTEndpoint() string {
+	scheme := "http"
```

Comment:
Should probably return `""` if `mms.RESTPort <= 0`? So that we show no http endpoint in the status when the rest proxy is disabled

Reply:
Good catch, updated